### ISSUE TITLE
FontAwesome 4.4 Doesn't Have Balance Icon

### ISSUE THREAD

**Issue by osolano** [PrideChung/FontAwesomeKit#72]
Hello, 

In the new FontAwesome 4.4 set, there is a balance-scale icon that was introduced but it is not available in your kit.
http://fontawesome.io/icon/balance-scale/
Could you add this?

Regards,
Oliver

**Comment by top2tech** [comment]
Actually the following icons are also missing:

commentingOIconWithSize
diamondIconWithSize
blackTieIconWithSize
userSecretIconWithSize
streetViewIconWithSize

I happen to use them, so I find it out, I believe there are more that are missing, looks like there is some serious bug somewhere?

**Comment by top2tech** [comment]
false alarm.

It's because Cocoapod is not updated.
I was using the library directly from this git, then I switched to cocoapod, then some of icons are missing, as cocoapod has not been updated for about 1 year.

After I switch back to git, it works fine.

@osolano , the balance-scale is there in this git, could you try if the same thing happened to you?

**Comment by osolano** [comment]
@top2tech  yup, it is there in git. I'm using git version now instead of cocoapods. thanks!

**Comment by osolano** [comment]
adjusted title since icons are only right now missing in latest cocoapods version 2.2.0

**Comment by RemyDCF** [comment]
If you want to use the latest version of FontAwesomeKit in Cocoapods, you should use
`pod 'FontAwesomeKit', git: 'https://github.com/PrideChung/FontAwesomeKit.git'`
